Subject: [PATCH] Fix configure failure for --disable-spdy

The @HAVE_OPENSSL@ conditional is used in src/Makefile.in so when SPDY
support is forcibly disabled the AM_CONDITIONAL is never executed, causing
configure to bail out.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura libmicrohttpd-0.9.29.orig/configure.ac libmicrohttpd-0.9.29/configure.ac
--- libmicrohttpd-0.9.29.orig/configure.ac	2013-09-02 06:17:44.660620584 -0300
+++ libmicrohttpd-0.9.29/configure.ac	2013-09-02 06:36:44.598189751 -0300
@@ -341,6 +341,8 @@
                    enable_spdy="yes"], 
                    [AM_CONDITIONAL(HAVE_OPENSSL, false)
                    enable_spdy="no"])
+else
+ AM_CONDITIONAL(HAVE_OPENSSL, false)
 fi
 
 if test "$enable_spdy" = "yes"
